src/lib/i18n/LanguageKeys/Commands/Sokoban.ts

```diff
@@ -0,0 +1,20 @@
+import { FT, T, type Value } from '@skyra/http-framework-i18n';
+
+// Root
+export const RootName = T('commands/sokoban:name');
+export const RootDescription = T('commands/sokoban:description');
+
+// Options
+export const PlayLevel = 'commands/sokoban:optionLevel';
+export const OptionChooseLevel = 'commands/sokoban:optionChooseLevel';
+export const CustomLevel = 'commands/sokoban:optionCustomLevel';
+export const OptionImportCustomLevel = 'commands/sokoban:optionImportCustomLevel';
```

Comment:
Please separate subcommands and options, and update the keys in the i18n files too